src/android/StatusBar.java

```diff
@@ -38,6 +38,8 @@
 
 public class StatusBar extends CordovaPlugin {
     private static final String TAG = "StatusBar";
+    private String _bgColor;
```

Comment:
IMO these variables should not begin with an underscore

Reply:
My habits leaked into the PR, even though it's not consistent with Apache's codebase :grin:

I'll remove the underscores when I come back to this task.